src/marqo/tensor_search/tensor_search.py

```diff
@@ -776,12 +811,17 @@ def search(config: Config, index_name: str, text: str, result_count: int = 3, hi
         raise errors.InvalidArgError(f"Search called with unknown search method: {search_method}")
     
     if reranker is not None:
+        # SEARCH TIMER-LOGGER (reranking)
+        start_rerank_time = timer()
         rerank.rerank_search_results(search_result=search_result, query=text, 
                     model_name=reranker, device=config.indexing_device, 
                 searchable_attributes=searchable_attributes, num_highlights=1 if simplified_format else num_highlights)
+        end_rerank_time = timer()
+        total_rerank_time = end_rerank_time - start_rerank_time
+        logger.info(f"search ({search_method.lower()}) reranking: took {(total_rerank_time):.3f}s to rerank results.")
 
-    time_taken = datetime.datetime.now() - t0
-    search_result["processingTimeMs"] = round(time_taken.total_seconds() * 1000)
+    time_taken = timer() - t0
+    search_result["processingTimeMs"] = round(time_taken * 1000)
     search_result["query"] = text
```

Comment:
Can we also log the total time for search (along with search-type)?

Reply:
done